scripts/webidl/convert.js

```diff
@@ -30,6 +37,104 @@ const transformer = new Webidl2js({
     return `
       return ${identifier}(globalObject, interfaceName, new.target);
     `;
+  },
+  // https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflecting-content-attributes-in-idl-attributes
+  processReflect(idl, implObj) {
+    const reflectAttr = idl.extAttrs.find(attr => attr.name === "Reflect");
+    const attrName = (reflectAttr && reflectAttr.rhs && reflectAttr.rhs.value.replace(/_/g, "-")) || idl.name.toLowerCase();
+
+    if (idl.extAttrs.find(attr => attr.name === "ReflectURL")) {
+      // Allow DOMString also due to https://github.com/whatwg/html/issues/5241.
+      if (!isSimpleIDLType(idl.idlType, "USVString") && !isSimpleIDLType(idl.idlType, "DOMString")) {
+        throw new Error("[ReflectURL] specified on non-USVString attribute");
+      }
+      const parseURLToResultingURLRecord =
+        this.addImport("../helpers/document-base-url", "parseURLToResultingURLRecord");
+      const serializeURL = this.addImport("whatwg-url", "serializeURL");
+      return {
+        get: `
+          const value = ${implObj}.getAttributeNS(null, "${attrName}");
+          if (value === null) {
+            return "";
+          }
+          const urlRecord = ${parseURLToResultingURLRecord}(value, ${implObj}._ownerDocument);
+          if (urlRecord !== null) {
+            return ${serializeURL}(urlRecord);
+          }
+          return conversions.USVString(value);
+        `,
+        set: `
+          ${implObj}.setAttributeNS(null, "${attrName}", V);
+        `
+      };
+    }
+
+    if (isSimpleIDLType(idl.idlType, "DOMString") || isSimpleIDLType(idl.idlType, "USVString")) {
+      const isUSV = isSimpleIDLType(idl.idlType, "USVString");
+      return {
+        get: `
+          const value = ${implObj}.getAttributeNS(null, "${attrName}");
+          return value === null ? "" : ${isUSV ? "conversions.USVString" : ""}(value);
```

Comment:
```suggestion
          return value === null ? "" : ${isUSV ? "conversions.USVString(value)" : "value"};
```

(a bit easier to tell what's going on, IMO.)

Reply:
Done.